Domains/4-Application/ISM/IsmSynchronization.ecschema.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ECSchema schemaName="IsmSynchronization" alias="ismsync" description="Schema for synchronizing Ism and iModel entities" version="01.00.02" displayLabel="Ism Synchronization" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="BisCore" version="01.00.00" alias="bis" />
+    <ECSchemaReference name="StructuralAnalysis" version="01.00.00" alias="sa" />
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
+        </ProductionStatus>
+        <SchemaLayerInfo xmlns="BisCustomAttributes.01.00.00">
+            <Value>Application</Value>
+        </SchemaLayerInfo>
+    </ECCustomAttributes>
+
+    <ECEntityClass typeName="IsmAspect" modifier="Sealed" displayLabel="Ism Aspect" description="This Aspect stores Ism class instance Id for synchronization with iModel class instances." >
+        <BaseClass>bis:ElementUniqueAspect</BaseClass>
+        <ECProperty propertyName="IsmId" typeName="string" displayLabel="IsmId" description="Id of an Ism class instance" />
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="ElementOwnsIsmAspect" strength="embedding" modifier="Sealed">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="bis:Element"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="IsmAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="ReactionAspect" modifier="Sealed" displayLabel="Reaction Aspect" description="This aspect shows if element is a reaction element." >
+        <BaseClass>bis:ElementUniqueAspect</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="LoadOwnsReactionAspect" strength="embedding" modifier="Sealed">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="sa:Load"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="ReactionAspect"/>
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IsmCurveMemberAspect" modifier="Sealed" displayLabel="Ism Calculated Curve Member Properties Aspect" description="This aspect stores calculated properties of Ism Curve Member." >
+        <BaseClass>bis:ElementUniqueAspect</BaseClass>
+        <ECProperty propertyName="Length" typeName="double" displayLabel="Length" description="Length of Ism Curve Member." />
```

Comment:
I've added most of the missing kind of quantities.
- I did not add any for dimensionless quantities for ratios, frictions, etc. Do we want to have a NONE unit in AecUnits schema?
- I did not add any unit for AngularFriction (1/pressure) and WobbleFriction (1/length) as AecUnits schema did not have any values for those

Reply:
IMO, the Aecu schema captures "typical" KOQ used with default formats. However, some shared-domains/applications/connectors may have the need to specify more appropriate formats for their KOQs, or introduce "specialized" KOQs that are not really "typical" across AEC. The later happens a lot in "analytical" schemas, which usually bring units associated to coefficients and equations specific to a specific kind of analysis.

That point of view drove me to introduce separate unit-related schemas for my efforts in shared-Civil and the Civil Connector:
* https://github.com/iTwin/bis-schemas/blob/master/Domains/2-DisciplinePhysical/Civil/RoadRailUnits/RoadRailUnits.ecschema.xml
* https://github.com/iTwin/bis-schemas/blob/master/Domains/4-Application/Connectors/CivilInfrastructureFramework/CifUnits.ecschema.xml

We can discuss it further with the BWG.

Regarding dimensionless quantities, ideally semantically-rich KOQs should be used/introduced when possible. E.g. a "ratio" may mean "slope" (vertical meters over horizontal meters) or "how smooth a transition between two road curves is" (back -curve radius over following-curve radius). We don't want to use the same KOQ for both cases.

Having said that, there are cases where the semantics behind a "ratio" unit are unknown - that is more typical on Connector/dynamic schemas. A generic "ratio" KOQ is the only option in that case - but we don't want to introduce such a KOQ in the Aecu schema, otherwise it will be abused (i.e. nobody will introduce the more semantically-rich "ratio" KOQs).